src/ReactApp.tsx

```diff
@@ -104,7 +104,7 @@ export default function ReactApp(settings: any) {
     const { isGamificationOn } = settings.settings;
 
     result = array.map((item, index) => 
-      <div style={{ display: 'flex', flexDirection: 'row', justifyContent: 'space-between' }} key={ index } className={'item'} onClick={(event) => handleItemClick(event, item) }>
+      <div style={{ display: 'flex', flexDirection: 'row', justifyContent: 'space-between', marginLeft: 'clamp(5px, 5px, 10px)', marginRight: 'clamp(5px, 9px, 10px)' }} key={ index } className={'item'} onClick={(event) => handleItemClick(event, item) }>
```

Comment:
Please move this to a new pull request that's specifically for the "Make the dashboard look good no matter the theme" ticket